lightning/src/ln/channelmanager.rs

```diff
@@ -3479,7 +3489,7 @@ impl<ChanSigner: ChannelKeys, M: Deref + Sync + Send, T: Deref + Sync + Send, K:
 			}
 		} else {
```

Comment:
About the zero-value channel id branch, wouldn't be better to sanitize early that case at `open_channel` reception ? Such value would be a hint of either our CSRNG being burst of the counterparty being buggy or sneaky. Either way we should discard that value from our internal channel map ?

Reply:
Hmm? Not sure exactly what you mean - zero-value channel id means "close all channels with me", its not a specific reference to a channel.